### PR TITLE
# 670  Added links to footer, header, and landing page

### PR DESCRIPTION
**Pull request by shailikadakia-oicr**
#### `Links` Component
As part of the `Footer`, updated the URL for:
  - `About OHCRN`
  - `Help Centre`
  - `Contact`
  - `Terms and Conditions`
  - `Privacy Policy`

#### `Footer` Component
Added to the `Footer Dictionary` to add a link to OICR. Removed logic for OICR route based on current language as the Footer dictionary for EN and FR can route to different links. **Can revert this change by adding:** 

` const oicrHref = currentLang === 'fr' ? `${textDict.oicrUrl}/${currentLang}` : `${textDict.oicrUrl}`;

The div containing the Instagram and Twitter logos has `style={{ display: 'none'}} `

#### `Header` Component
- Added to the `Header Dictionary` to add a link to OICR from the logo
- Added a prop to `HelpButton` to pass along the link from `Header`

#### `SideImageLayout` Component
- Added the url prop to `HelpButton`

#### `Home` Component
- Added to the `Landing Page Dictionary` to add a link to OHCRN About Us
- Note: initially structured change by adding an env variable but have refactored code to remove that addition 

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [x] Tests are updated (if required) and passing
- [ ] [Axe DevTools](https://www.deque.com/axe/browser-extensions/) "Full Page Scan" of updated pages passes without introducing new Critical or Serious issues (UI changes only)
- [ ] PR feedback has been addressed **if applicable**
- [ ] New environment variables added to `.env.schema` files, `README.md`
- [ ] Added copyrights to new files

### CODE REVIEW

**Comment by mistryrn** [comment]
Heads up, looks like there's a build error related to the changes made to the `HelpButton` component, need to update the `SideImageLayout` as well:
Screenshot 2024-09-11 at 4 37 53 PM

(From https://jenkins.sweng-dev.cumulus.genomeinformatics.org/blue/organizations/jenkins/softeng%2Fohcrn%2Fapp-builds%2Fplatform/detail/feat%2F670-add-support-links/3/pipeline)

**Comment by shailikadakia-oicr** [comment]
Thank you @mistryrn I've added the prop type to the component and will check the build to see if it was successful

File: apps/consent-ui/src/components/common/Footer/Links.tsx [hunk 1]
```diff
@@ -28,31 +28,37 @@ import styles from './Footer.module.scss';
...
 	return (
 		<div className={clsx(styles.linkGrid, styles.links)}>
 			{footerLinks.map((link) => (
-				<Link className={styles.link} key={link.translationKey} href={link.url}>
+				<Link
+					className={styles.link}
+					key={link.translationKey}
+					href={link.url}
+					target="_blank"
+					prefetch={false}
```
Comment (mistryrn) [inline apps/consent-ui/src/components/common/Footer/Links.tsx:60]: Thanks for adding this, too 👏

File: apps/consent-ui/src/components/common/Header/Header.tsx [hunk 2]
```diff
@@ -127,7 +127,7 @@ const Header = ({ currentLang, textDict, languageToggleProps }: HeaderContentPro
 					</div>
 					{/* TODO: implement real help button, ticket TBD */}
```
Comment (mistryrn) [inline apps/consent-ui/src/components/common/Header/Header.tsx:128]: Can remove this comment now 🙂

File: apps/consent-ui/src/components/common/Header/HelpButton.tsx [hunk 3]
```diff
@@ -1,21 +1,27 @@
...
-const HelpButton = ({ label }: { label: string }) => {
+const HelpButton = ({ label, url }: { label: string; url: string }) => {
 	return (
-		<Button
+		<LinkButton
 			variant="secondary"
 			color="green"
 			className={styles.helpButton}
 			LeftIcon={<Help />}
-			onClick={() => console.log('HALP')}
```
Comment (mistryrn) [inline apps/consent-ui/src/components/common/Header/HelpButton.tsx:15]: #rip `HALP` 2023-2024 😢 🫡 🪦

File: apps/consent-ui/src/config/appConfig.ts [hunk 4]
```diff
@@ -39,6 +39,7 @@ const getAppConfig = (): AppConfig => ({
 		defaultAppConfig.FEATURE_CONSENT_PDF_BUTTONS,
 	OHCRN_EMAIL: process.env.OHCRN_EMAIL || defaultAppConfig.OHCRN_EMAIL,
 	OHCRN_HOME_LINK: process.env.OHCRN_HOME_LINK || defaultAppConfig.OHCRN_HOME_LINK,
+	OHCRN_ABOUT_LINK: process.env.OHCRN_ABOUT_LINK || defaultAppConfig.OHCRN_ABOUT_LINK,
```
Comment (mistryrn) [inline apps/consent-ui/src/config/appConfig.ts:42]: Does this need to come from an env var? Let's confirm with Brandon tomorrow, but I suspect we can just pull this from the dictionary instead.

**Review by mistryrn — COMMENTED** [review]
Looks good! Great PR description and thanks for cleaning up some additional things that were previously in the codebase as well 🙂 

Just had one question about the use of env vars for the "About Us" link – let's confirm that with Brandon tomorrow and proceed from there.

**Pull request merged**